marketplaceItems/sitPoint/appResources/appData/sitClient.js

```diff
@@ -129,6 +132,7 @@
     var overlays = {
```

Comment:
A general comment that starts here: Please do not use the style of putting functions and members inside vars. It makes code harder to read/parse, even though it does make collapsing easier. It also means putting variables near where they're first used/relevant is impossible.

Please make function/variable names verbose and global to the file. If you want to split data/functions into individual chunks, please use modules.

Reply:
I'm going to not review the rest of the file until this is taken care of, since I know changing this will change a lot of other aspects of the code.